openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -78,32 +87,38 @@ def to_representation(self, user):
             # https://docs.djangoproject.com/en/1.8/ref/databases/#fractional-seconds-support-for-time-and-datetime-fields
             "date_joined": user.date_joined.replace(microsecond=0),
             "is_active": user.is_active,
-            "bio": AccountLegacyProfileSerializer.convert_empty_to_None(profile.bio),
-            "country": AccountLegacyProfileSerializer.convert_empty_to_None(profile.country.code),
+            "bio": AccountLegacyProfileSerializer.convert_empty_to_None(profile.bio) if profile else None,
```

Comment:
Instead of appending the `or None` statements multiple time, it would be easier to read if you created a default `data` object with key's value set to None, then if the profile exists, set the values on the keys.